var/spack/repos/builtin/packages/hip/package.py

```diff
@@ -214,7 +219,13 @@ class Hip(CMakePackage):
     patch(
         "0011-Improve-compilation-without-git-repo-and-remove-compiler-rt-linkage-for-host"
         ".5.0.2.patch",
-        when="@5.0.2:",
+        when="@5.0.2:5.1.3",
+    )
+
+    patch(
+        "0012-Improve-compilation-without-git-repo-and-remove-compiler-rt-linkage-for-host"
+        ".5.2.0.patch",
+        when="@5.2.0:",
```

Comment:
Same comments as above:
1. It would be better is this is fixed upstream
2. In any case a comment explaining what the patch does, why is it needed and where it was taken from is welcome to help the next reader of `package.py`

Reply:
@alalazo  I have been working with the team to get this changes into the next release . While 5.2.1 will not have this change, i would see that this is addressed in 5.3.0 release.

Reply:
The compilation without git was fixed in https://github.com/ROCm-Developer-Tools/hipamd/commit/56b32604729cca08bdcf00c7a69da8a75cc95b8a and is on track for ROCm 5.3. If you have a fix for the rt linkage, that's great!